index.html

```diff
@@ -43,6 +36,174 @@ <h1 class="h1-green">
       <i class="fab fa-github"></i>
       <i class="fab fa-angellist"></i>
     </nav>
+    <nav>
+      <img src="images/Action.svg" alt="Action icon">
+    </nav>
   </header>
+  <section class="works-container">
+    <header class="works-header">
+      <h1 class="h1-blue">
+        My Recent<br> Works
+      </h1>
```

Comment:
- [ ] [OPTIONAL] Every HTML document should have only one `h1` element in the entire file. Since you used an `h1` at line 23, other headers in the document should be from `h2` down the ladder. I highly recommend you fix this to enhance the application's SEO rating.